test/autests/gold_tests/http2_abort/replay_files/client_goaway_after_headers.yaml

```diff
@@ -0,0 +1,80 @@
+meta:
```

Comment:
We should add the Apache license for replay files. I think I've missed that in other replay files... we should add that with a separate commit. Don't worry for this PR. But for this new file, let's add the license.